stable/mysql/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.configurationFiles }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+data:
+{{- range $key, $val := .Values.configurationFiles }}
+  {{ $key }}: |-
```

Comment:
Just for clarification: If the actual value already as the `|-` _(as mentioned in the example)_ would this not create a problem?